src/main/java/com/leverx/employeestat/rest/service/impl/ReportServiceImpl.java

```diff
@@ -32,45 +53,111 @@ public ReportServiceImpl(WorkRepository workRepository) {
     }
 
     @Override
-    public void export(HttpServletResponse response) {
-        createHeaders();
-        try (ServletOutputStream outputStream = response.getOutputStream()) {
+    @Scheduled(cron = "0 0 0 1 * *")
```

Comment:
as you already created application.properties, cron also can be extracted to properties and default cron once a month can be provided via SpEL in case of cron not provided in properties